### ISSUE TITLE
Error message when activating dense redstone ore on servers

### ISSUE THREAD

**Issue by CalaMariGold** [codetaylor/pyrotech-1.12#350]
All information here: https://github.com/Rebirth-of-the-Night/Rebirth-Of-The-Night/issues/557

**Comment by codetaylor** [comment]
I'm unable to reproduce this and the error log provided indicates a fairly generic Minecraft failure with no actionable data.

**Comment by codetaylor** [comment]
Scratch that. Reproduced and here's the full error:

```
[11:19:02] [Client thread/FATAL] [minecraft/Minecraft]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_251]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_251]
	at net.minecraft.util.Util.runTask(Util.java:54) [Util.class:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1178) [Minecraft.class:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:442) [Minecraft.class:?]
	at net.minecraft.client.main.Main.main(Main.java:118) [Main.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_251]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_251]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_251]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_251]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_251]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_251]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_251]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_251]
	at net.minecraftforge.gradle.GradleStartCommon.launch(GradleStartCommon.java:97) [start/:?]
	at GradleStart.main(GradleStart.java:25) [start/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.client.audio.PositionedSoundRecord.<init>(PositionedSoundRecord.java:45) ~[PositionedSoundRecord.class:?]
	at net.minecraft.client.audio.PositionedSoundRecord.<init>(PositionedSoundRecord.java:40) ~[PositionedSoundRecord.class:?]
	at net.minecraft.client.multiplayer.WorldClient.playSound(WorldClient.java:509) ~[WorldClient.class:?]
	at net.minecraft.client.multiplayer.WorldClient.playSound(WorldClient.java:497) ~[WorldClient.class:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleSoundEffect(NetHandlerPlayClient.java:1828) ~[NetHandlerPlayClient.class:?]
	at net.minecraft.network.play.server.SPacketSoundEffect.processPacket(SPacketSoundEffect.java:78) ~[SPacketSoundEffect.class:?]
	at net.minecraft.network.play.server.SPacketSoundEffect.processPacket(SPacketSoundEffect.java:13) ~[SPacketSoundEffect.class:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:21) ~[PacketThreadUtil$1.class:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_251]
	at java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:266) ~[?:1.8.0_251]
	at java.util.concurrent.FutureTask.run(FutureTask.java) ~[?:1.8.0_251]
	at net.minecraft.util.Util.runTask(Util.java:53) ~[Util.class:?]
	... 15 more
```

**Comment by codetaylor** [comment]
This is fixed in dev and will be released in the next patch.

**Comment by codetaylor** [comment]
Fixed in `1.5.10`
